tasks/user.yml

```diff
@@ -82,14 +82,15 @@
 
 - name: create .ssh directory for {{ user.name }}
   file:
-    path: "{{ user.home }}/.ssh"
+    path: "{{ item.home }}/.ssh"
```

Comment:
The looped variable is renamed to `user` in `tasks/main.yml`: (See the `loop_var` parameter.)

```yaml
- name: loop over users_user_list
  include: user.yml
  with_items:
    - "{{ users_user_list }}"
  loop_control:
    label: "{{ user.name }}"
    loop_var: user
  when:
    - users_user_list is defined
```

Can you tell me what issue you ran into? I'm pretty sure you are having a legitimate issue with this role.

Reply:
Hey. The condition "user.home is defined" is never fulfilled. As a result, the task "create .ssh directory for {{user.name}}" will be skipped.

Reply:
Ah, I think I understand what you mean, and indeed you are right. Let me have a look if the role needs a slight refactor for this.

Reply:
Use the ansible playbook debugger to verify this (type "p task_vars [" user "] [" home "]" in the debugger console).